puppetdb/Dockerfile

```diff
@@ -1,18 +1,36 @@
-# Determines source "edge" or binary "release" builds
-ARG build_type=release
-ARG UBUNTU_CODENAME=jammy
-# NOTE: never pass as a build-arg / must match .dockerenv -- used in logback.xml
-ARG LOGDIR=/opt/puppetlabs/server/data/puppetdb/logs
+FROM ubuntu:24.04
```

Comment:
is there any benefit in switching to 24.04 now? Or can we wait until puppet provides proper packages?

Reply:
its mainly about cves in the base os..

Noble 24.04
>  ✔ Scanned for vulnerabilities     [37 vulnerability matches]  
>   ├── by severity: 0 critical, 1 high, 14 medium, 16 low, 6 negligible

Jammy 22.04
> ✔ Scanned for vulnerabilities     [74 vulnerability matches]  
>   ├── by severity: 0 critical, 1 high, 27 medium, 39 low, 7 negligible